frontend/src/modules/notification/models.ts

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from "react"
+import { AnyAction } from "redux"
+
+export const ALERT = 'ALERT'
+export const ALERT_REMOVE = `${ALERT}.REMOVE`
+
+export enum AlertType {
+    SUCCESS = 'success',
+    INFO = 'info',
+    WARNING = 'warning',
+    ERROR = 'error'
+}
+
+export interface AlertProps {
```

Comment:
We should have two kinds of alerts - alerts that stay open until the user dismisses them (like we have now) and alerts that stay open for a number of seconds and then are closed automatically. That's useful for info alerts like "Your changes were submitted successfully" which are tiresome to close manually every time they pop up.

Reply:
the existing solution is to set duration to 0 to represent infinity (until user closes it). what do you think of that?